docs/minikube.md

```diff
@@ -0,0 +1,106 @@
+## Minikube
+
+Draft requires a wildcard domain and an ingress controller. Here how to set everything
+up when using [Minikube][Minikube]
+
+### Setup the certificate
```

Comment:
This should not be required unless you're setting it up via kube-lego. Did something change in the default packs while I was away?